ntc_templates/templates/cisco_ios_show_boot.textfsm

```diff
@@ -40,12 +40,12 @@ Start
   ^Load\s+for\s+
   ^Time\s+source\s+is
   ^Boot\s*Mode\s*=\s*${BOOT_MODE}
-  ^Boot\soptimization\s:\s${BOOT_OPTIMIZATION}
+  ^Boot\s*optimization\s*:\s*${BOOT_OPTIMIZATION}
```

Comment:
```suggestion
  ^Boot\s+optimization\s+:\s+${BOOT_OPTIMIZATION}
```

Reply:
Would just like to have the minimum one space caught.

Reply:
Thanks that also works